guides/theming.md

```diff
@@ -50,7 +50,7 @@ Finally, if your app's content **is not** placed inside of a `md-sidenav-contain
 need to add the `mat-app-background` class to your wrapper element (for example the `body`). This
 ensures that the proper theme background is applied to your page.
 
-### Defining a custom theme
+### Step 1: Defining a custom theme
```

Comment:
Say "Step 1" doesn't really fit here because it's not in the context of anything that came before it. A better approach would be 
```markdown
### Defining a custom theme

#### Create an scss file for your theme
...
#### Include the generated styles in your application
...
```